server/http/validation/variable.go

```diff
@@ -0,0 +1,143 @@
+package validation
```

Comment:
What's the idea behind having this module be part of a `validation` subfolder? is this something we can continue to replicate for future scenarios?

Reply:
Yes. One thing that the backend has to do better is to validate the input. I added this module there and I want to keep adding some validation objects there. `http` already has quite some objects there and I didn't want to make it bigger.